tests/delete/tests.py

```diff
@@ -582,3 +596,10 @@ def test_fast_delete_empty_no_update_can_self_select(self):
                 User.objects.filter(avatar__desc='missing').delete(),
                 (0, {'delete.User': 0})
             )
+
+    @skipIfDBFeature("can_defer_constraint_checks")
```

Comment:
Should be skipUnless (or maybe even no skip at all), so that it has a chance to run on Postgres, correct? I would take this to be what Mariusz was referring to.